src/System.Private.CoreLib/shared/System/SpanHelpers.T.cs

```diff
@@ -53,6 +50,65 @@ public static int IndexOf<T>(ref T searchSpace, int searchSpaceLength, ref T val
             return -1;
         }
 
+        // Adapted from IndexOf(...)
+        public static unsafe bool Contains<T>(ref T searchSpace, T value, int length)
```

Comment:
Is there any benefit in marking this as `in T value`? My guess is 'no' due to (at the least) consistency, but I am interested in the answer regardless. My understand of `in` is that if `T` was a `struct`, we'd save a copy here?

Reply:
> My guess is 'no' due to (at the least) consistency

Right.

`in` is double edged sword. It may improve or regress performance depending on number of factors. There is good answer what is universally better here.